app/views/shared/_cost_of_living.html.slim

```diff
@@ -3,6 +3,10 @@
 
 = govuk_details(summary_text:)
   p.govuk-body = t("cost_of_living_details.text")
-  p.govuk-body
-    - t("cost_of_living_details.list#{suffix}").each
-      div = _1
+  - if FeatureFlags.enabled?(:mtr_accelerated, without_session_data: true)
+    ul
```

Comment:
we can get rid of the `ul`, as it adds an additional space on the top which we don't want.